features/FxA-54-push-on-status-change/README.md

```diff
@@ -30,6 +30,9 @@ As a concrete metric
 we can measure the number of calls to `/recovery_email/status`
 and check that it decreases as this feature rolls out.
 
+We will also add additional metrics to signal when Desktop verifies 
+via Push.
```

Comment:
Actually, for the `with the expectation that this will be a fairly high percentage.` I think we need to tweak the metrics a bit to only include verifications within ~1 hour. We want to have a high percentage of push verifications for engaged users. We won't be able to get a high percentage for push because verifications take > 30 hours on average